proposed/0006-Launcher.md

```diff
@@ -0,0 +1,204 @@
+- Feature Name: SAFE-Launcher
+- Type: New Product
+- Related components: [safe_client](https://github.com/maidsafe/safe_client), [safe_nfs](https://github.com/maidsafe/safe_nfs)
+- Start Date: 11-September-2015
+
+# Summary
+
+Launcher will be a gateway for any App that wants to work on the SAFE-Network on a user's behalf.
+
+# Motivation
+
+## Why?
+
+App's access of the SAFE-Network on behalf of the user is an issue with high security concerns. Further, decentralisation allows for uniformity in experience on various machines concerning user installed Apps.
```

Comment:
I think it would be good to expand a bit on the security concerns (a couple of examples maybe).

Also, isn't another motivating factor UX, in that apps won't need to get the user's credentials from the user in order to access the network?